earth_enterprise/rpms/opengee-server/post-install.sh

```diff
@@ -0,0 +1,231 @@
+    sed -i "s/IA_GEAPACHE_USER/$GEAPACHEUSER/" /opt/google/gehttpd/conf/gehttpd.conf
+    sed -i "s/IA_GEPGUSER/$GEPGUSER/" /opt/google/gehttpd/conf/gehttpd.conf
+    sed -i "s/IA_GEGROUP/$GEGROUP/" /opt/google/gehttpd/conf/gehttpd.conf
+
+    # c) Create a new file ‘/etc/opt/google/fusion_server_version’ and
+    # add the below text to it.
+    echo "$RPM_PACKAGE_VERSION" >/etc/opt/google/fusion_server_version
+
+    # d) Create a new file ‘/etc/init.d/gevars.sh’ and prepend the below lines.
+    echo -e "GEAPACHEUSER=$GEAPACHEUSER\nGEPGUSER=$GEPGUSER\nGEFUSIONUSER=$GEFUSIONUSER\nGEGROUP=$GEGROUP" >$BININSTALLROOTDIR/gevars.sh
+}
+
+# this too probably should be done inside the rpm...
+fix_postinstall_filepermissions()
+{
+    # PostGres
+    chown -R $GEPGUSER:$GEGROUP $BASEINSTALLDIR_VAR/pgsql/
+
+    # Apache
+    mkdir -p $BASEINSTALLDIR_OPT/gehttpd/conf.d/virtual_servers/runtime
+    chmod -R 755 $BASEINSTALLDIR_OPT/gehttpd
+    chmod -R 775 $BASEINSTALLDIR_OPT/gehttpd/conf.d/virtual_servers/runtime/
+    chown -R $GEAPACHEUSER:$GEGROUP $BASEINSTALLDIR_OPT/gehttpd/conf.d/virtual_servers/
+    chown -R $GEAPACHEUSER:$GEGROUP $BASEINSTALLDIR_OPT/gehttpd/htdocs/cutter/
+    chmod -R 700 $BASEINSTALLDIR_OPT/gehttpd/htdocs/cutter/globes/
+    chown $GEAPACHEUSER:$GEGROUP $BASEINSTALLDIR_OPT/gehttpd/htdocs/.htaccess
+    chown -R $GEAPACHEUSER:$GEGROUP $BASEINSTALLDIR_OPT/gehttpd/logs
+
+    # Publish Root
+    chmod 775 $PUBLISHER_ROOT/stream_space
+    # TODO - Not Found
+    # chmod 644 $PUBLISHER_ROOT/stream_space/.config
+    chmod 644 $PUBLISHER_ROOT/.config
+    chmod 755 $PUBLISHER_ROOT
+    chown -R $GEAPACHEUSER:$GEGROUP $PUBLISHER_ROOT/stream_space
+    chown -R $GEAPACHEUSER:$GEGROUP $PUBLISHER_ROOT/search_space
+
+    # Run and logs ownership
+    chown root:$GEGROUP $BASEINSTALLDIR_OPT/run
+    chown root:$GEGROUP $BASEINSTALLDIR_VAR/run
+    chown root:$GEGROUP $BASEINSTALLDIR_VAR/log
+    chown root:$GEGROUP $BASEINSTALLDIR_OPT/log
+
+    # setuid requirements...hmmm
+    chmod +s /opt/google/bin/gerestartapache /opt/google/bin/geresetpgdb /opt/google/bin/geserveradmin
+
+    # Tutorial and Share
+    find /opt/google/share -type d -exec chmod 755 {} \;
+    find /opt/google/share -type f -exec chmod 644 {} \;
+    chmod ugo+x /opt/google/share/searchexample/searchexample
+    chmod ugo+x /opt/google/share/geplaces/geplaces
+    chmod ugo+x /opt/google/share/support/geecheck/geecheck.pl
+    chmod ugo+x /opt/google/share/support/geecheck/convert_to_kml.pl
+    chmod ugo+x /opt/google/share/support/geecheck/find_terrain_pixel.pl
+    chmod ugo+x /opt/google/share/support/geecheck/pg_check.pl
+    # Note: this is installed in install_fusion.sh, but needs setting here too.
+    #TODO
+    # chmod ugo+x $BASEINSTALLDIR_OPT/share/tutorials/fusion/download_tutorial.sh
+    # this should already be true...
+    chown -R root:root /opt/google/share
+
+    #TODO
```

Comment:
same as above

Reply:
This looks like something that should be fixed, but, that can be done on a clearer ticket. I'm getting rid of it.